packages/tools/client-debugger/client-debugger-view/src/components/AudienceStateTable.tsx

```diff
@@ -33,6 +35,9 @@ export interface AudienceStateTableProps {
 export function AudienceStateTable(props: AudienceStateTableProps): React.ReactElement {
 	const { audienceStateItems } = props;
```

Comment:
Style-wise, I would recommend looking at @alexvy86's PR and use the same info icons he added there: #15157

Reply:
Changed using the `Info16Regular`. Realized that this ruins the alignment between the icon and the text. Will fix this too.